docs/coffee.html

```diff
@@ -0,0 +1,432 @@
+                    fill="#403F3D" />
+                  <path
+                    d="M29.1181 51.0909C29.1181 51.7178 29.0007 52.2548 28.7658 52.7017C28.5328 53.1468 28.2147 53.4877 27.8113 53.7245C27.4097 53.9612 26.9542 54.0796 26.4448 54.0796C25.9353 54.0796 25.4789 53.9612 25.0755 53.7245C24.6739 53.4858 24.3558 53.144 24.1209 52.6989C23.888 52.2519 23.7715 51.7159 23.7715 51.0909C23.7715 50.464 23.888 49.9281 24.1209 49.483C24.3558 49.036 24.6739 48.6942 25.0755 48.4574C25.4789 48.2207 25.9353 48.1023 26.4448 48.1023C26.9542 48.1023 27.4097 48.2207 27.8113 48.4574C28.2147 48.6942 28.5328 49.036 28.7658 49.483C29.0007 49.9281 29.1181 50.464 29.1181 51.0909ZM28.0584 51.0909C28.0584 50.6496 27.9893 50.2775 27.851 49.9745C27.7147 49.6695 27.5253 49.4394 27.2828 49.2841C27.0404 49.1269 26.7611 49.0483 26.4448 49.0483C26.1285 49.0483 25.8491 49.1269 25.6067 49.2841C25.3643 49.4394 25.1739 49.6695 25.0357 49.9745C24.8993 50.2775 24.8311 50.6496 24.8311 51.0909C24.8311 51.5322 24.8993 51.9053 25.0357 52.2102C25.1739 52.5133 25.3643 52.7434 25.6067 52.9006C25.8491 53.0559 26.1285 53.1335 26.4448 53.1335C26.7611 53.1335 27.0404 53.0559 27.2828 52.9006C27.5253 52.7434 27.7147 52.5133 27.851 52.2102C27.9893 51.9053 28.0584 51.5322 28.0584 51.0909Z"
+                    fill="#403F3D" />
+                  <path
+                    d="M22.5536 50.1449H21.4911C21.4608 49.9707 21.4049 49.8163 21.3235 49.6818C21.2421 49.5455 21.1407 49.4299 21.0195 49.3352C20.8983 49.2406 20.7601 49.1695 20.6048 49.1222C20.4513 49.0729 20.2856 49.0483 20.1076 49.0483C19.7913 49.0483 19.511 49.1279 19.2667 49.287C19.0224 49.4442 18.8311 49.6752 18.6928 49.9801C18.5546 50.2832 18.4854 50.6534 18.4854 51.0909C18.4854 51.536 18.5546 51.911 18.6928 52.2159C18.833 52.519 19.0243 52.7481 19.2667 52.9034C19.511 53.0568 19.7904 53.1335 20.1048 53.1335C20.279 53.1335 20.4419 53.1108 20.5934 53.0654C20.7468 53.018 20.8841 52.9489 21.0053 52.858C21.1284 52.7671 21.2317 52.6553 21.315 52.5227C21.4002 52.3902 21.4589 52.2387 21.4911 52.0682L22.5536 52.0739C22.5138 52.3504 22.4277 52.6099 22.2951 52.8523C22.1644 53.0947 21.993 53.3087 21.7809 53.4943C21.5688 53.6781 21.3207 53.822 21.0366 53.9262C20.7525 54.0284 20.4371 54.0796 20.0906 54.0796C19.5792 54.0796 19.1228 53.9612 18.7212 53.7245C18.3197 53.4877 18.0034 53.1459 17.7724 52.6989C17.5413 52.2519 17.4258 51.7159 17.4258 51.0909C17.4258 50.464 17.5423 49.9281 17.7752 49.483C18.0082 49.036 18.3254 48.6942 18.7269 48.4574C19.1284 48.2207 19.583 48.1023 20.0906 48.1023C20.4144 48.1023 20.7156 48.1477 20.994 48.2387C21.2724 48.3296 21.5205 48.4631 21.7383 48.6392C21.9561 48.8135 22.1351 49.0275 22.2752 49.2813C22.4173 49.5332 22.5101 49.821 22.5536 50.1449Z"
+                    fill="#403F3D" />
+                </svg>
+              </a>
+            </div>
+            <div class="nav">
+              <ul class="nav list">
+                <li class="nav-list">
```

Comment:
nav list and nav-list could be a bit misleading perhaps. A common name to add is "item", so nav-item or list-item

Reply:
Fixed